caringcaribou/modules/test.py

```diff
@@ -14,5 +14,5 @@ def module_main(_):
     """Runs all Caring Caribou unit tests"""
     print_interface_header()
     # Run tests
-    test_suite = unittest.TestLoader().discover("./tests")
+    test_suite = unittest.TestLoader().discover("./caringcaribou/tests")
```

Comment:
This only works if one runs `$ cc.py -i vcan0 test` from the root of the `caringcaribou` folder, as the path is relative. Do you have any idea on how to make this load the tests from the right folder, regardless of where `cc.py` is run? (through some standard package magic mechanism, I suppose 😅)

Current behavior when run from the root of the `caringcaribou` folder (just as expected):
```
~/Desktop/caringcaribou$ cc.py -i vcan0 test

----------------------
CARING CARIBOU v0.4dev
----------------------

Loading module 'test'

Running tests using CAN interface 'vcan0'

test_create_iso_14229_1 (test_iso_14229_1.DiagnosticsOverIsoTpTestCase) ... ok
test_ecu_reset_failure_on_invalid_reset_type (test_iso_14229_1.DiagnosticsOverIsoTpTestCase) ... ok
<etc, cropped for brevity>
```

Current (unexpected) behavior when run anywhere else:
```
~/Desktop$ cc.py -i vcan0 test

----------------------
CARING CARIBOU v0.4dev
----------------------

Loading module 'test'

Running tests using CAN interface 'vcan0'

Traceback (most recent call last):
  File "/usr/local/bin/cc.py", line 11, in <module>
    load_entry_point('caringcaribou==0.4.dev0', 'console_scripts', 'cc.py')()
  File "/usr/local/lib/python3.8/dist-packages/caringcaribou-0.4.dev0-py3.8.egg/caringcaribou/caringcaribou.py", line 116, in main
    cc_mod.module_main(args.module_args)
  File "/usr/local/lib/python3.8/dist-packages/caringcaribou-0.4.dev0-py3.8.egg/caringcaribou/modules/test.py", line 17, in module_main
    test_suite = unittest.TestLoader().discover("./caringcaribou/tests")
  File "/usr/lib/python3.8/unittest/loader.py", line 346, in discover
    raise ImportError('Start directory is not importable: %r' % start_dir)
ImportError: Start directory is not importable: './caringcaribou/tests'
```

Reply:
I think it's fixed in the last push, test module now discovers unit tests from `caringcaribou.tests`.